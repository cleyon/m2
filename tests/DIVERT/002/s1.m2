@sequence count
@newcmd header{text}
@divert 9
@sequence count next
(in stream 9) count=@count@, text=@text@
@divert
@endcmd
@header{First Header}
@header{Second Header}
@header{Third Header}
Table of Contents
@undivert 9
All done
