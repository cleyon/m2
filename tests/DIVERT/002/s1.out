Table of Contents
(in stream 9) count=1, text=First Header
(in stream 9) count=2, text=Second Header
(in stream 9) count=3, text=Third Header
All done
